<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcelOperatorClass</class>
 <widget class="QMainWindow" name="ExcelOperatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pictures Paster</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/refresh-32.png</normaloff>Resources/refresh-32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 255, 238);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ExcutePaster">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileBrowse">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PicDirEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>select a picture directory</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExcelPathEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseExcel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>select the target Excel file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Press the Button below to Start</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Adobe Gothic Std B&quot;;
color: rgb(144, 229, 255);</string>
       </property>
       <property name="text">
        <string>Step 1:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Adobe Gothic Std B&quot;;
color: rgb(144, 229, 255);</string>
       </property>
       <property name="text">
        <string>Step 2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;Adobe Gothic Std B&quot;;
color: rgb(144, 229, 255);</string>
       </property>
       <property name="text">
        <string>Step 3:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="WaittingView" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 230, 213);
border-color: rgb(255, 255, 127);</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 18pt &quot;Adobe Devanagari&quot;;</string>
     </property>
     <property name="text">
      <string>Please wait for a while...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="RowEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>13</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Item No starts at</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row ,</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ColumnEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>231</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="ApactRatioLock">
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;Berlin Sans FB&quot;;</string>
       </property>
       <property name="text">
        <string>Lock pictures' Aspect Ratio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Pw_Cw">
       <property name="text">
        <string>Pictures' width = Cells' width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Ph_Ch">
       <property name="text">
        <string>Pictures' height = Cells' height</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>ExcutePaster</zorder>
   <zorder>FileBrowse</zorder>
   <zorder>PicDirEdit</zorder>
   <zorder>label</zorder>
   <zorder>ExcelPathEdit</zorder>
   <zorder>browseExcel</zorder>
   <zorder>label_2</zorder>
   <zorder>line_2</zorder>
   <zorder>line</zorder>
   <zorder>label_6</zorder>
   <zorder>RowEdit</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>ColumnEdit</zorder>
   <zorder>label_11</zorder>
   <zorder>label_10</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>WaittingView</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="exceloperator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExcutePaster</sender>
   <signal>clicked()</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>slotExecute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileBrowse</sender>
   <signal>clicked()</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>slotBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseExcel</sender>
   <signal>clicked()</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>slotBrowseExcelTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ApactRatioLock</sender>
   <signal>clicked(bool)</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>setLockAspectRatio(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Pw_Cw</sender>
   <signal>clicked(bool)</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>setFixedWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ph_Ch</sender>
   <signal>clicked()</signal>
   <receiver>ExcelOperatorClass</receiver>
   <slot>setFixedHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotExecute()</slot>
  <slot>slotBrowse()</slot>
  <slot>slotBrowseExcelTarget()</slot>
  <slot>setFixedHeight()</slot>
  <slot>setFixedWidth()</slot>
  <slot>setLockAspectRatio(bool)</slot>
 </slots>
</ui>
